src/integ_test_resources/ios/sdk/integration/cdk/custom_resources/iot_custom_authorizerUserPass_provider/README.md

```diff
@@ -0,0 +1,11 @@
+## IoT Custom Authorizer
```

Comment:
What about the "enhanced" part? This looks like the regular custom authorizer behavior.

Reply:
Sorry, will review the text and update this, thanks!